config.go

```diff
@@ -131,6 +131,81 @@ func (cfg *Config) SetConsumeFuel(enabled bool) {
 	runtime.KeepAlive(cfg)
 }
 
+// SetTailCall configures whether tail calls are enabled
+func (cfg *Config) SetTailCall(enabled bool) {
+	C.wasmtime_config_wasm_tail_call_set(cfg.ptr(), C.bool(enabled))
+	runtime.KeepAlive(cfg)
+}
+
+// SetFunctionReferences configures whether function references are enabled
+func (cfg *Config) SetFunctionReferences(enabled bool) {
+	C.wasmtime_config_wasm_function_references_set(cfg.ptr(), C.bool(enabled))
+	runtime.KeepAlive(cfg)
+}
+
+// SetGC configures whether GC is enabled
+func (cfg *Config) SetGC(enabled bool) {
+	C.wasmtime_config_wasm_gc_set(cfg.ptr(), C.bool(enabled))
+	runtime.KeepAlive(cfg)
+}
+
+// SetWideArithmetic configures whether wide arithmetic is enabled
+func (cfg *Config) SetWideArithmetic(enabled bool) {
+	C.wasmtime_config_wasm_wide_arithmetic_set(cfg.ptr(), C.bool(enabled))
+	runtime.KeepAlive(cfg)
+}
```

Comment:
Mind putting `Wasm` in the name (e.g. `SetWasmGC`) for these? (to match the C API)

Reply:
Ah, good point